### ISSUE TITLE
twitter integration

### ISSUE THREAD

**Issue by larshp** [dotabap/dotabap-validator#21]
https://twitter.com/dotabap

algorithm:
1: for each entry in the json list
2: check if there is an existing tweet for it
3: if no existing tweet, tweet it

@tricktresor

**Comment by tricktresor** [comment]
Wouldn‘t it be easier if there was a form on dotabap.org to add a new project?
In this form could be an option to twitter about the project.
Maybe simply add a twitter link to every project so that everyone could twitter about the project (kudos, suggestions, questions, ...)

**Comment by larshp** [comment]
there is no database, everything is static pages served by github, updated daily via travis

the list is: https://github.com/dotabap/dotabap-list/blob/master/list.json

**Comment by tricktresor** [comment]
Yes. Got that. Isn‘t it possible to a add an entry to the list programatically?

**Comment by tricktresor** [comment]
https://developer.github.com/v3/repos/contents/#update-a-file

**Comment by larshp** [comment]
yeah, but it requires a key, which basically means anyone can change the file to anything
